src/ExtensionsSample/Program.cs

```diff
@@ -59,20 +75,22 @@ public static void Main(string[] args)
             // Add or remove types from this list to choose which functions will 
             // be indexed by the JobHost.
             config.TypeLocator = new SamplesTypeLocator(
-                typeof(ErrorMonitoringSamples),
-                typeof(FileSamples),
-                typeof(MiscellaneousSamples),
-                typeof(SampleSamples),
-                typeof(SendGridSamples),
-                typeof(TableSamples),
-                typeof(TimerSamples),
-                typeof(WebHookSamples));
-
-            host.Call(typeof(MiscellaneousSamples).GetMethod("ExecutionContext"));
-            host.Call(typeof(FileSamples).GetMethod("ReadWrite"));
-            host.Call(typeof(SampleSamples).GetMethod("Sample_BindToStream"));
-            host.Call(typeof(SampleSamples).GetMethod("Sample_BindToString"));
-            host.Call(typeof(TableSamples).GetMethod("CustomBinding"));
+                //typeof(ErrorMonitoringSamples),
+                //typeof(FileSamples),
+                //typeof(MiscellaneousSamples),
+                //typeof(SampleSamples),
+                //typeof(SendGridSamples),
+                //typeof(TableSamples),
+                //typeof(TimerSamples),
+                //typeof(WebHookSamples),
+                typeof(ApiHubSamples));
+
+            //host.Call(typeof(MiscellaneousSamples).GetMethod("ExecutionContext"));
+            //host.Call(typeof(FileSamples).GetMethod("ReadWrite"));
+            //host.Call(typeof(SampleSamples).GetMethod("Sample_BindToStream"));
+            //host.Call(typeof(SampleSamples).GetMethod("Sample_BindToString"));
+            //host.Call(typeof(TableSamples).GetMethod("CustomBinding"));
+            host.Call(typeof(ApiHubSamples).GetMethod("Writer"));
```

Comment:
What are the config requirements for running the ApiHub samples? I think you should follow the same model as the DocumentDb / EasyTables samples - include instructions in the sample on how to add to the type locator to run, but don't include by default.